Emby.Server.Implementations/Library/LibraryManager.cs

```diff
@@ -1232,6 +1228,20 @@ public BaseItem GetItemById(Guid id)
             return item;
         }
 
+        /// <inheritdoc />
+        public BaseItem GetItemById(Guid id, Guid userId)
+        {
+            var user = userId.IsEmpty() ? null : _userManager.GetUserById(userId);
+            return GetItemById(id, user);
+        }
+
+        /// <inheritdoc />
+        public BaseItem GetItemById(Guid id, User user)
```

Comment:
are these overloads needed?

Reply:
Not needed, but I figured since we sometimes have the user object we might as well use it and not attempt to retrieve it again

Reply:
I am talking about the generic and BaseItem specific overloads. Isn't the generic sufficient?

Reply:
Ah- the BaseItem specific already existed and the generic was recently added.
I continued adding the BaseItem specific to stay the pattern but it would be simple enough to remove and update the callers